skaffold/src/main/kotlin/com/google/container/tools/skaffold/run/SkaffoldFilesComboBox.kt

```diff
@@ -51,9 +53,23 @@ class SkaffoldFilesComboBox : JComboBox<VirtualFile>() {
      * the file, adds it and selects it.
      */
     fun setSelectedSkaffoldFile(skaffoldFile: VirtualFile) {
-        skaffoldFilesMutableModel.addElement(skaffoldFile)
+        var existingElement = false
+        for (i in 0..skaffoldFilesMutableModel.size) {
+            if (skaffoldFilesMutableModel.getElementAt(i) == skaffoldFile) {
```

Comment:
does VirtualFile implement `equals`?

Reply:
A bit hard to find particular implementation, there are many of them, but the javadoc claims the following:
```
* A particular file is represented by equal {@code VirtualFile} instances for the entire lifetime of the IntelliJ IDEA process, unless the file
 * is deleted, in which case {@link #isValid()} will return {@code false}.
 * <p/>
 * VirtualFile instances are created on request, so there can be several instances corresponding to the same file.
 * All of them are equal, have the same hashCode and use shared storage for all related data, including user data (see {@link UserDataHolder}).
```
Last sentence says it should work.

Reply:
cool